sample_data/users.json

```diff
@@ -176,7 +152,6 @@
     "userType": "USER",
     "joinedOrganizations": [
       "6537904485008f171cf29924",
-      "6637904485008f171cf29924",
```

Comment:
1. Restore all these `6637904485008f171cf29924` entries
2. We don't want any existing functionality to be changed in importing sample data

Reply:
This is still in there but now it has become the `default organization`. You can find it in `sample_data/defaultOrganization.json`.

Reply:
OK, is the default organization part of the sample data import too?

Reply:
> OK, is the default organization part of the sample data import too?

We discussed it in this PR, it will only be imported if there are no organizations in database!

Reply:
1. Please restore the original functionality with the four sample organizations.
2. We don't want any of the other functionality modified for the PR

Reply:
Revert all changes to this file. This is out of scope